lib/linguist/heuristics.rb

```diff
@@ -127,6 +127,14 @@ def call(data)
         Language["ECL"]
       end
     end
+    
+    disambiguate ".es" do |data|
+      if /^\s*(?:%%|main\s*\(.*?\)\s*->)/.match(data)
+        Language["Erlang"]
+      elsif /(?:\/\/|("|')use strict\1|export\s+default\s|\/\*.*?\*\/)/m.match(data)
```

Comment:
For the `\/\*.*?\*\/` part, it won't match if the comment is on several lines. Would `\/\*` be sufficient?
Also, would a `^\s*var\s` make sense? `var` is very common in JavaScript.